pages/storage_and_backup/object_storage/Versioning_Doc/guide.en-gb.md

```diff
@@ -0,0 +1,129 @@
+---
+title: 'Getting Started with Versioning Using OVHcloud APIs'
+excerpt: 'Learn how to enable and manage versioning for your OVHcloud Object Storage buckets using APIs. Future updates will cover the OVHcloud Control Panel.'
```

Comment:
The 2nd sentence should be withdrawn from the excerpt and inserted as a "primary" in the guide's body. Same for the FR version.